README.md

```diff
@@ -169,134 +137,116 @@ Some changes do not require an RFC:
-'ready' and may be implemented with the goal of eventual inclusion into
+In short, to get a major feature added to CDK/JSII, one usually writes an RFC 
+as a markdown file and gets it approved and merged into the RFC repo. At that point the RFC is
+'approved' and may be implemented with the goal of eventual inclusion into
 CDK/JSII.
 
-- [Create a tracking issue](https://github.com/awslabs/aws-cdk-rfcs/issues/new?template=tracking-issue.md)
+1. [Create a **tracking issue**](https://github.com/awslabs/aws-cdk-rfcs/issues/new?template=tracking-issue.md)
   for the proposed feature if one doesn't already exist. Use the tracking issue
   template as a guide. If a tracking issue already exists, make sure to update
   it and assign it to let others know you're working on a proposal.
-- Fork the [RFC repo](https://github.com/awslabs/aws-cdk-rfcs).
-- Copy `0000-template.md` to `text/<rfc#>-<my-feature>.md` where <rfc#> is the
+2. Fork the [RFC repo](https://github.com/awslabs/aws-cdk-rfcs).
+3. Copy `0000-template.md` to `text/<rfc#>-<my-feature>.md` where <rfc#> is the
   tracking issue number and `<my-feature>` is the rfc title.
-- Fill in the RFC. Put care into the details: **We welcome all honest efforts to
+4. Fill in the RFC. Put care into the details: **We welcome all honest efforts to
   contribute.**.
-- Submit a pull request with the title `RFC: <rfc#> <title>` where <rfc#> is the
+5. Submit a **pull request** with the title `RFC: ### <title>` where ### is the
   tracking issue number and title is the name of the proposal. As a pull request
   the RFC will receive design feedback from the core team and the larger
   community, and the author should be prepared to make revisions in response.
-  - Update the tracking issue with a link to the RFC PR.
-- Build consensus and integrate feedback. RFCs that have broad support are much
-  more likely to make progress than those that don't receive any comments.
-- Eventually, the team will decide whether the RFC is a candidate for inclusion
+6. Update the tracking issue with a link to the RFC PR.
+7. **Advertise** your RFC amongst stakeholders via social channels (e.g. twitter)
+  and your team. Build consensus and integrate feedback. RFCs that have broad support 
+  are much more likely to make progress than those that don't receive any comments.
+8. Eventually, the team will decide whether the RFC is a candidate for inclusion
   in CDK/JSII.
-- RFCs that are candidates for inclusion in CDK/JSII will enter a "final comment
-  period" lasting 3 calendar days. The beginning of this period will be signaled
+9. RFCs that are candidates for inclusion in CDK/JSII will enter a "**final comment
+  period**" lasting 3 calendar days. The beginning of this period will be signaled
   by a team member adding a comment and label on the RFCs pull request.
-- An RFC can be modified based upon feedback from the team and community.
+10. An RFC can be modified based upon feedback from the team and community.
```

Comment:
Is this specifically during the final comment period, or at any time (e.g., after approval)?

Reply:
It can be modified at any time. If it is modified after it was merged, a new PR will be needed (I'll add).